sql/pre_install/tables.sql

```diff
@@ -466,23 +466,19 @@ CREATE TABLE _timescaledb_catalog.compression_algorithm (
   CONSTRAINT compression_algorithm_pkey PRIMARY KEY (id)
 );
 
-CREATE TABLE _timescaledb_catalog.hypertable_compression (
-  hypertable_id integer NOT NULL,
-  attname name NOT NULL,
-  compression_algorithm_id smallint,
-  segmentby_column_index smallint,
-  orderby_column_index smallint,
-  orderby_asc boolean,
-  orderby_nullsfirst boolean,
-  -- table constraints
-  CONSTRAINT hypertable_compression_pkey PRIMARY KEY (hypertable_id, attname),
-  CONSTRAINT hypertable_compression_hypertable_id_orderby_column_index_key UNIQUE (hypertable_id, orderby_column_index),
-  CONSTRAINT hypertable_compression_hypertable_id_segmentby_column_index_key UNIQUE (hypertable_id, segmentby_column_index),
-  CONSTRAINT hypertable_compression_compression_algorithm_id_fkey FOREIGN KEY (compression_algorithm_id) REFERENCES _timescaledb_catalog.compression_algorithm (id),
-  CONSTRAINT hypertable_compression_hypertable_id_fkey FOREIGN KEY (hypertable_id) REFERENCES _timescaledb_catalog.hypertable (id) ON DELETE CASCADE
+CREATE TABLE _timescaledb_catalog.compression_settings (
+	relid regclass NOT NULL,
```

Comment:
Some questions about this:
- is it guaranteed to survive dump/restore? (the oids might change)
- for chunks, we're going to use the uncompressed chunk oid here? The compressed ones are often recreated, not sure if the uncompressed oid can change as well